src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -1002,5 +1002,50 @@ describe('ReactDOMFiber', () => {
         container
       );
     });
+
+    describe('disableNewFiberFeatures', () => {
+      var ReactFeatureFlags = require('ReactFeatureFlags');
```

Comment:
We're already pretty inconsistent throughout the codebase :D One day we'll go through and clean this up